test/CMakeLists.txt

```diff
@@ -3,21 +3,18 @@
 ###############
 
 LIST(APPEND axl_test_srcs
-    test_axl.c
-    test_axl_sync.c
-    test_axl_async_daemon.c
-    test_axl_async_ibmbb.c
+    axl_cp
```

Comment:
Is the `.c` needed on the end of this?

Reply:
It builds for me without it, which seems weird.  I'll add it in